schemas/styles/csl.rnc

```diff
@@ -283,24 +283,31 @@ div {
     }
   term.attributes =
     (attribute name { terms },
-     [ a:defaultValue = "long" ] attribute form { term.form }?)
+     [ a:defaultValue = "long" ] attribute form { term.form }?,
+     term.types)
     | (attribute name { terms.ordinals },
        attribute form { "long" }?,
        attribute gender-form { "masculine" | "feminine" }?,
        attribute match {
          "last-digit" | "last-two-digits" | "whole-number"
-       }?)
+       }?,
+       term.types)
     | (attribute name { terms.long-ordinals },
        attribute form { "long" }?,
        attribute gender-form { "masculine" | "feminine" })
     | (attribute name { terms.gender-assignable },
        attribute form { "long" }?,
-       attribute gender { "masculine" | "feminine" })
+       attribute gender { "masculine" | "feminine" },
+       term.types)
   
   ## "verb-short" reverts to "verb" if the "verb-short" form is not available.
   ## "symbol" reverts to "short" if the "symbol" form is not available.
   ## "verb" and "short" revert to "long" if the specified form is not available.
   term.form = "long" | "verb" | "short" | "verb-short" | "symbol"
+  term.types = 
+    attribute types {
```

Comment:
```suggestion
    attribute for-types {
```

Reply:
Also, don't you need to add a docstring here?

Reply:
I like this. LGTM to merge after that.

Reply:
docstring is there now